App/templates/calculadora.html

```diff
@@ -1,14 +1,49 @@
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>App Calculator</title>
-</head>
-<body>
-    <div class = "container">
-        <h1>Escoge la operación</h1>
+<html>
+  <head>
+    <link
+      rel="stylesheet"
+      href="{{ url_for('static', filename ='css/estilos.css')}}"
+    />
+  </head>
+
+  <body onload="init();">
+    <div>
+      <a href="{{ url_for('api') }}" class="regresar"><</a>
```

Comment:
`class` attribute should be before, then `href `

Reply:
fixed